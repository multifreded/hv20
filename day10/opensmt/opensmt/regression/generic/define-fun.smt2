(define-fun foo ((a integer) (b real) (c frobbozian)) ((_ bv0 1 2 3 4) q q1 q2 q3) (let ((x 1) (y 2)) bar))
