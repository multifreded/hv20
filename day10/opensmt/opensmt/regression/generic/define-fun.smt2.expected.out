(error "Illegal command before set-logic: define-fun")

